devices/flash-zynq7000/flashdrv.c

```diff
@@ -174,10 +189,11 @@ static int flashdrv_sectorErase(addr_t offs)
 	if (offs > CFI_SIZE_FLASH(cfi->chipSize))
 		return -EINVAL;
 
-	if ((i = flashdrv_regionFind(offs)) < 0)
-		return i;
+	res = flashdrv_regionFind(offs, &id);
+	if (id < 0)
+		return res;
```

Comment:
```suggestion
	if (res < 0)
		return res;
```
`id` is of type `u32` and always `>= 0`.
see also: [PR](https://github.com/phoenix-rtos/phoenix-rtos-devices/pull/173)

Reply:
Yeah, I totally agree with you, stupid mistake from my side. Thanks for pointing this out.